lib/string-checker.js

```diff
@@ -130,7 +130,12 @@ StringChecker.prototype = {
 
         this._configuredRules.forEach(function(rule) {
             errors.setCurrentRule(rule.getOptionName());
-            rule.check(file, errors);
+
+            try {
```

Comment:
I can debate it for a long time, which would lead nowhere, so i really don't want to argue about that, if you ask "if that is a real problem" - yes, i think it is, if we can do something about that, i would prefer to do it.